FDW - Release branch (#3331)
EnterpriseDB/docs: product_docs/docs/mongo_data_adapter/5/mongo_rel_notes/mongo5.5.0_rel_notes.mdx

---
title: "Version 5.5.0"
---

Enhancements, bug fixes, and other changes in MongoDB Foreign Data Wrapper 5.5.0 
include:

| Type        | Description                                                                                                                                                                                                                                                 |
| ----------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |

> | Feature     | Push down `ORDER BY` to remote MongoDB servers: If possible push the `ORDER BY` clause to the remote MongoDB server. This approach provides the ordered result set from the foreign server, which can help to have an efficient merge join.    |

Reviewer: ```suggestion
| Feature     | When possible, we push the `ORDER BY` clause to the remote MongoDB server. This approach provides the ordered result set from the foreign server, which can help to have an efficient merge join.    |
```

> | Feature     | Push down `LIMIT`/`OFFSET` to remote MongoDB servers: Where possible, perform `LIMIT` and `OFFSET` operations on the remote server. This reduces network traffic between local Postgres and remote MongoDB servers.                            |

Reviewer: ```suggestion
| Feature     | When possible, perform `LIMIT` and `OFFSET` operations on the remote server. This reduces network traffic between local Postgres and remote MongoDB servers.                            |
```

> | Enhancement | Improved the WHERE clause pushdown to remote MongoDB servers: Now supports recursive operator expressions, Boolean expressions, relabel types, and vars on both sides of an operator.                                                          |

Reviewer: ```suggestion
| Enhancement | Improved the WHERE clause pushdown to remote MongoDB servers. Now supports recursive operator expressions, Boolean expressions, relabel types, and vars on both sides of an operator.                                                          |
```
